Src/Couchbase.Linq/Mapping/DocIdAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Couchbase.Linq.Mapping
+{
+    /// <summary>
+    /// Provides a means of mapping a field to a Couchbase document key or Id.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    public class DocIdAttribute : Attribute
```

Comment:
Jeff,

I think you may have accidentally left this old file in the pull, unless you were planning on using it for something else.

Brant